### ISSUE TITLE
Compatibility with react-redux@7 and next@9

### ISSUE THREAD

**Issue by mmiszy** [danielr18/connected-next-router#23]
Hi! Is this package compatible with react-redux@7 and next@9? Thank you :)

**Comment by danielr18** [comment]
Yes, I'll be releasing a new version to fix the peer dependency warnings very soon.

**Comment by adonig** [comment]
Does react-redux@7 with next@9 really work? I followed the instructions in the README and get

```
index.js:1 Warning: Failed context type: The context `store` is marked as required in `WithConnectedRouter`, but its value is `undefined`.
    in WithConnectedRouter (at _app.js:36)
    in Provider (at _app.js:35)
    in Container (at _app.js:34)
    in MyApp (created by withRedux(MyApp))
    in withRedux(MyApp)
    in Suspense (created by AppContainer)
    in Container (created by AppContainer)
    in AppContainer
```

**Comment by danielr18** [comment]
@adonig Are you using version 1? If version 1 isn't working, I'm sure `2.0.0-alpha.1` will, but I haven't documented that version yet.

**Comment by shkarupa-alex** [comment]
Same error and i don't see version 2.0.0-alpha.1 anywhere

**Comment by mmiszy** [comment]
@shkarupa-alex https://www.npmjs.com/package/connected-next-router/v/2.0.0-alpha.1

**Comment by getorca** [comment]
even with v2.0.0-alpha.1 I'm getting the Warning

```Warning: Failed context type: The context `store` is marked as required in `Component`, but its value is `undefined`.
    in WithConnectedRouter (at _app.js:51)
    in Context.Provider
    in Provider (at _app.js:50)
    in Container (at _app.js:49)
    in MyApp
    in ConnectFunction
    in ConnectFunction
    in withRedux(Connect(MyApp))
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in AppContainer

**Comment by danielr18** [comment]
I would need a minimal repro to check that, I don't get that warning in the example: https://github.com/danielr18/connected-next-router/tree/typescript/examples/basic

**Comment by danielr18** [comment]
Example is also working with v1 of this library, Next.js 9 and react-redux 7, it's just that it has peer dependencies warnings. Do you have any chance of creating a repo where I can take a look or codesandbox?

**Comment by danielr18** [comment]
However, `WithConnectedRouter` is only in v0 of the library, so you might not be using `v2.0.0-alpha.1`.

**Comment by danielr18** [comment]
I've updated the `latest` tag in npm to point to 1.0.2, so if you are using react-redux 6 or 7, you can just `npm i connected-next-router`.

**Comment by adonig** [comment]
@danielr18 Version 1.0.2 works. Thank you 😊🙏
